functions.php

```diff
@@ -261,7 +261,18 @@ function getTotalCol($question,$project_id,$question_1,$conditionDate,$topScoreM
         }
     }
 
-    $missingOverall += $missing_col;
+    $row_questions_1 = ProjectData::getRowQuestionsParticipantPerception();
+    $RecordSetOverallMissing = \REDCap::getData($project_id, 'array', $recordIds, null, null, null, false, false, false, "[".$question_1."] = ''".$conditionDate);
```

Comment:
Should this getData() call include a field list so that all data is not pulled from the DB?

Reply:
Good point. I just changed it :)
I also found an unused variable